src/transformers/testing_utils.py

```diff
@@ -25,18 +25,18 @@
 from pathlib import Path
 
 from .file_utils import (
-    _datasets_available,
-    _faiss_available,
-    _flax_available,
-    _pandas_available,
-    _scatter_available,
-    _sentencepiece_available,
-    _tf_available,
-    _tokenizers_available,
-    _torch_available,
-    _torch_tpu_available,
+    is_datasets_available,
```

Comment:
All changes in test files are to remove the use of private variables that don't exist anymore, to their public function counterpart.